README.md

```diff
@@ -63,51 +38,34 @@ $ ./flockeropenssl/generate_flocker_certs.sh -h
 -h | --help (This help message)
 ```
 
-Example
+Examples:
 
 ```
-./flockeropenssl/generate_flocker_certs.sh -d="ec2-52-91-11-106.compute-1.amazonaws.com" -n="ec2-52-91-11-106.compute-1.amazonaws.com,node2,node3" -f=/etc/flocker/ssl/flockeropenssl/openssl.cnf
+./flockeropenssl/generate_flocker_certs.sh -d=www.foobar.com -k=1024 -c=staging-1 -n=one,two
 ```
-
-Example Output
 ```
-0:ec2-52-91-11-106.compute-1.amazonaws.com
-1:node2
-2:node3
-Generating RSA private key, 4096 bit long modulus
-.....................................................................++
-....................................++
-e is 65537 (0x10001)
-Generating RSA private key, 4096 bit long modulus
-..............................................................................++
-.
-.
-. (cut out long output)
+./flockeropenssl/generate_flocker_certs.sh -d="ec2-52-91-11-106.compute-1.amazonaws.com" -n="ec2-52-91-11-106.compute-1.amazonaws.com,node2,node3" -f=/etc/flocker/ssl/flockeropenssl/openssl.cnf
```

Comment:
Did you mean to keep this in here? I also see an example above `./flockeropenssl/generate_flocker_certs.sh -d=www.foobar.com -k=1024 -c=staging-1 -n=one,two` we should probably just keep that one.